app/models/server/models/Sessions.tests.js

```diff
@@ -246,7 +246,7 @@ describe('Sessions Aggregates', () => {
 	if (!process.env.MONGO_URL) {
 		before(function() {
 			this.timeout(120000);
-			return mongoUnit.start({ version: '3.2.22' })
+			return mongoUnit.start({ version: '4.0.22' })
```

Comment:
we should use the older mongodb supported version for our unit tests, which currently is 3.4.x .. so we cannot use 4.0 here :/